src/resources/locales/en.json

```diff
@@ -81,7 +81,7 @@
         "moreInfo": "Book Details",
         "myBooks": "My books",
         "noPublicationHelpL1": "Drop a publication here",
-        "noPublicationHelpL2": "or use the \"+\" button above.",
+        "noPublicationHelpL2": "or use the \"import publication\" button above.",
```

Comment:
some thoughts: string interpolation with the existing already-translated label would (1) avoid duplication / risk of divergence during GUI updates, (2) avoid creating a TODO for translators ... in the flip side, this forces us to replace the old "+" with the new interpolated string in all locales.

What do you think?